install.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -euo pipefail || exit 1
+
+log() {
+	echo "$@" >&2
+}
+
+if [[ $HOSTTYPE != "x86_64" ]]; then
+	log "arch $HOSTTYPE is not supported"
+	log "please see https://sail.dev/docs/installation"
+	exit 1
+fi
+
+if ! command -v curl > /dev/null && ! command -v wget > /dev/null; then
+	log "please install curl or wget to use this script"
+	exit 1
+fi
+
+download() {
+	if command -v curl > /dev/null; then
+		curl --progress-bar -L "$1"
+	elif command -v wget > /dev/null; then
+		wget "$1" -O -
+	fi
+}
+
+latestReleaseURL() {
+	log "finding latest release"
+	local os=$1
+	download https://api.github.com/repos/cdr/sail/releases/latest |
+		jq -r ".assets[]
```

Comment:
I don't think it's a good idea to assume `jq` is installed.

Reply:
That's a fair point. It just makes it so nice and easy 😭

Reply:
Jank shell solution:
```
curl https://api.github.com/repos/cdr/sail/releases/latest | grep "browser_download_url" | grep "linux" | awk -F": " '{print $2}'
```

Reply:
yeah definitely can't use jq...